packages/page-mixer/src/index.tsx

```diff
@@ -49,10 +63,13 @@ const PageSwap: FC = () => {
         <LiquidityInformation />
       </Col>
       <Col span={24}>
-        <DepositConsole />
+        <WithdrawConsole />
       </Col>
     </Row>
   );
 };
 
-export default PageSwap;
+export default pageWithFeatures({
+  features: ['mixer'],
+  message: "Mixer isn't supported on the current cain ,please consider change the current network",
```

Comment:
"The mixer module is not supported on the current chain, please change the current network."